site/content/en/docs/tasks/run/deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Run Deployment"
+linkTitle: "Deployment"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a Deployment as a Kueue-managed job.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
```

Comment:
I'd suggest providing a little more context.  For example,

Although Kueue does not yet support managing a Deployment as a single workload, it is still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the Deployment.